Camera2D.gd

```diff
@@ -0,0 +1,16 @@
+extends Camera2D
+
+
+const DEAD_ZONE: int = 140
+@onready var alpaca = $"../Alpaca"
+
+func _process(_delta) -> void:
+
+	self.position = alpaca.position
```

Comment:
This line makes the camera flickering. This is because it tries to go back to the position of the alpaco 60 times per second.
When you move the mouse is okay, due to _input, but if you stop moving it flickers